src/Security/InteractiveSecurityHelper.php

```diff
@@ -75,4 +77,22 @@ public function guessEntryPoint(SymfonyStyle $io, array $securityData, string $a
             current($authenticators)
         );
     }
+
+    public static function guessUserClass(SymfonyStyle $io, array $securityData): string
+    {
+        if (1 === \count($securityData['security']['providers']) && isset(current($securityData['security']['providers'])['entity'])) {
+            $entityProvider = current($securityData['security']['providers']);
+            $userClass = $entityProvider['entity']['class'];
+        } else {
+            $userClass = $io->ask(
+                'Enter the User class you want to authenticate (e.g. <fg=yellow>App\\Entity\\User</>)
+ (It has to be handled by one of the firewall\'s providers)',
+                class_exists('App\\Entity\\User') && isset(class_implements('App\\Entity\\User')[UserInterface::class]) ? 'App\\Entity\\User'
+                    : class_exists('App\\Security\\User') && isset(class_implements('App\\Security\\User')[UserInterface::class]) ? 'App\\Security\\User' : null,
```

Comment:
Oh, that's a nice idea! Let's move this logic into a private function though, and do it without ternaries. It'll make it much easier, because you can return quickly - e.g.

```php
if (class_exists('App\\Entity\\User') && isset(class_implements('App\\Entity\\User')[UserInterface::class])) {
    return 'App\\Entity\\User';
}

// same for other case
```